src/pages/page-account-complete.html

```diff
@@ -260,37 +261,7 @@
                     <img src="{{avatar}}">
                     <div class="info">
                         <div class="name">{{username}}</div>
-                        <div class="keybox">
-                            <template is="dom-if" if="{{!showPubKey}}">
-                                <div class="pubkey truncate" on-click="_togglePubKey">{{publicKey}}</div>
-                            </template>
-                            <template is="dom-if" if="{{showPubKey}}">
-                                <div class="pubkeybox">
-                                    <div class="pubkey key" title='click to select all'>{{publicKey}}</div>
-                                    <template is="dom-if" if="{{wide}}">
-                                        <template is="dom-if" if="{{!showPrivKey}}">
-                                            <div class="keyclose" on-click="_toggleKeys"></div>
-                                        </template>
-                                    </template>
-                                </div>
-                                <div class="privkeybox">
-                                    <template is="dom-if" if="{{!showPrivKey}}">
-                                        <span class="bluelink" on-click="_togglePrivKey">show my private key</span>
```

Comment:
This is the on clock we lost from above

Reply:
The problem is that it's coming out of a component now, so I'm gonna need you guys on this. I'm not sure how to handle it, other then fire it across the whole app, which might be a bit overkill?